pillar/vault/roles/xpro.sls

```diff
@@ -0,0 +1,26 @@
+vault:
+  roles:
+    xpro-app:
+      backend: postgresql-xpro
```

Comment:
The backend would be named differently, along the lines of `postgres-production-apps-xpro` based on the `orchestrate.aws.rds` script.